packages/rocketchat-ui/client/components/header/header.js

```diff
@@ -1,4 +1,4 @@
-/* globals fireGlobalEvent*/
+/* globals fireGlobalEvent, openRoom*/
```

Comment:
I don't like to use this function, but I've not found another public / exported library function which navigates to a room